tests/product/test_collect.py

```diff
@@ -294,3 +274,33 @@ def test_collect_logs_server_not_installed(self):
                            self.PA_ONLY_CLUSTER)
         self.upload_topology()
         self._assert_no_logs_downloaded()
+
+    def test_collect_logs_multiple_server_logs(self):
+        self.setup_cluster(NoHadoopBareImageProvider(),
+                           self.STANDALONE_PRESTO_CLUSTER)
+        self.run_prestoadmin('server start')
+        self.cluster.write_content_to_host('#!/bin/bash\n touch '
```

Comment:
That works.  I was trying to write to all the hosts, but it actually only matters than one host has multiple server logs.